README.md

```diff
@@ -1144,6 +1146,17 @@ rake chewy:journal:apply["$(date -v-1H -u +%FT%TZ)"] # apply journaled changes f
 rake chewy:journal:apply["$(date -v-1H -u +%FT%TZ)",users] # apply journaled changes for the past hour on UsersIndex only
 ```
 
+For the cases when journal has grown up to enormous size, classical way of deletion is not quite possible. Fortunately chewy internally uses [delete-by-query](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/docs-delete-by-query.html#docs-delete-by-query-task-api) ES function which support async execution with batching and [throttling](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-delete-by-query.html#docs-delete-by-query-throttle).
+
+Available options are listed below, they can by set by ENV variables:
+* `WAIT_FOR_COMPLETION` - boolean flag, options controls async execution, It waits by default, when set to `false`, Elasticsearch performs some preflight checks, launches the request, and returns a task you can use to cancel or get the status of the task
```

Comment:
```suggestion
* `WAIT_FOR_COMPLETION` - a boolean flag. It controls async execution. It waits by default. When set to `false`, Elasticsearch performs some preflight checks, launches the request, and returns a task you can use to cancel the task or get its status.
```

Reply:
> returns a task you can use to cancel the task or get its status.

maybe

> returns a special task you can use to cancel the deletion task or get the status of the deletion task.

I find a bit confusing to have two different tasks referred in the same sentence.

Reply:
Also, in the code we use `FALSE_VALUES = %w[0 f false off].freeze`, could we document here, how to set a `false` value?

Reply:
This is the same task. In the response you are getting task (reference - an id) that you can use to can use when requesting the task in Task API.